src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import './App.css'
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import HomePage from './components/homePage/HomePage';
+import MyAppointments from './components/appointmentInfo/MyAppointments';
+import DoctorList from './components/doctorInfo/DoctorList';
+import DeleteDoctor from './components/doctorInfo/DeleteDoctor';
+import PatientList from './components/forSuperUser/PatientList';
+import Appointment from './components/appointmentInfo/Appointment';
+import AppointmentList from './components/forSuperUser/AppointmentList';
+import UserList from './components/forSuperUser/UserList';
+import DoctorDetails from './components/doctorInfo/DoctorDetails.jsx';
+import ModalAppoint from './components/appointmentInfo/ModalAppoint';
+
+function App() {
```

Comment:
- [ ] OPTIONAL: Doctors still can't log in:

![image](https://github.com/baqar-abbas/appointment-app-front-end/assets/57421823/442af267-4a89-4ad6-b385-2a70da927477)

However, since this was not a requirement and you've nailed all the required functionality I'm marking this as optional.  

I still suggest taking a look at this before final evaluation. 💪